float32 angle
float32 distance_from_center_percent # 0.0 <= distance_from_center_percent <= 1.0
string emotion
